sru-scripts/manual/oracle-sru

```diff
@@ -17,13 +18,34 @@ assert_expected_vars PROPOSED_SCRIPT SAMPLE_CLOUDCONFIG LP_USER SSHOPTS \
 create_samplecloudconfig SAMPLE_CLOUDCONFIG
 create_setup_proposed_script
 
+function cleanup {
+    echo "Cleaning up instance ${INSTANCE_ID}"
+    oci compute instance terminate --force --instance-id="${INSTANCE_ID}"
+}
 
 echo "### BEGIN $UBUNTU_RELEASE"
 
-# TODO: Use the Oracle CLI
-echo "Please manually launch a $UBUNTU_RELEASE instance in Oracle and enter its IP address"
-echo "(Don't forget to add your SSH public key to the instance when you launch it.)"
-read -p "$UBUNTU_RELEASE IP address: " VM_IP
+declare -A releases
+releases=( ["xenial"]="16.04" ["bionic"]="18.04" ["focal"]="20.04" )
```

Comment:
Might also we worth a check:
if [ -z "${releases[${UBUNTU_RELEASE}]}" ]; then
 echo "Invalid release specified $UBUNTU_RELEASE"
 exit 1
fi